docs/metrics.md

```diff
@@ -0,0 +1,67 @@
+# Firecracker Metrics Configuration
+
+For the metrics capability, Firecracker uses a generic Metrics system.
+This system can be configured by sending a `PUT` API Request to the
+`/metrics` path.
+
+## Prerequisites
+
+In order to configure the Metrics, first you have to create the resource
+that will be used for storing the metrics:
+
+```bash
+# Create the required named pipe:
+mkfifo metrics.fifo
+
+# The Metrics system also works with usual files:
+touch metrics.file
+```
+
+## Configuring the system
+
+You can configure the Metrics system by sending the following API command:
+
+```bash
+curl --unix-socket /tmp/firecracker.socket -i \
+    -X PUT "http://localhost/metrics" \
+    -H "accept: application/json" \
+    -H "Content-Type: application/json" \
+    -d "{
+             \"metrics_path\": \"metrics.fifo\"
+    }"
+```
+
+Details about the required and optional fields can be found in the
+[swagger definition](../../src/api_server/swagger/firecracker.yaml).
+
+The `metrics.fifo` pipe will store the metrics in JSON format.
```

Comment:
nit: This should talk about the format in which we print metrics. Since it is not mandatory to use a pipe, we can write in a more generic way.
For example, we can say: The metrics are written to the `metrics_path` in JSON format.